### PR TITLE
LPS-179672 Add automations for Data Set and Client Extensions

### PR DESCRIPTION
**Pull request by JerryAdrianoo**
**Description:**
Add automations for edit option on Data Set Fields

**JIRA Ticket:**
https://liferay.atlassian.net/browse/LPS-179672

**Context about the problem with assert on apple emoji:**
I needed to add a @class in the @div from :apple_emoji: to differentiate from the others @divs. The reason is that apple emoji don't had anything to differentiate of others @divs on the same column. And considering that what was putted on code literally was one emoji, I couldn't test it on Poshi.
I asked from others devs how to do it, and this is the solution that I found.

### CODE REVIEW

**Comment by JerryAdrianoo** [comment]
ci:test:frontend-data-set

**Comment by JerryAdrianoo** [comment]
ci:test:frontend-data-set

**Comment by JerryAdrianoo** [comment]
ci:test:frontend-data-set

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/dependencies/liferay-sample-fds-cell-renderer.zip/static/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+var external_react_dom_x = y => { var x = {}; __webpack_require__.d(x, y); return x; }
+var external_react_dom_y = x => () => x
+const external_react_dom_namespaceObject = external_react_dom_x({ ["default"]: () => __WEBPACK_EXTERNAL_MODULE_react_dom_7dac9eee__["default"] });
+;// CONCATENATED MODULE: ./src/index.tsx
+
+
+const fdsCellRenderer = ({ value }) => {
+    const element = document.createElement('div');
+    const isGreen = value === 'Green';
+    if (isGreen) element.classList.add ("apple");
```
Comment (izaera) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/dependencies/liferay-sample-fds-cell-renderer.zip/static/index.js:46]: :ok_hand:

**Comment by JerryAdrianoo** [comment]
ci:test:frontend-data-set

**Review by CarlosBrichete — APPROVED** [review]

**Comment by CarlosBrichete** [comment]
LGTM

**Comment by JerryAdrianoo** [comment]
ci:test:frontend-data-set

**Comment by JerryAdrianoo** [comment]
ci:test:sf

**Comment by JerryAdrianoo** [comment]
ci:forward

**Pull request closed without merging**